app/views/govuk_component/docs/search.yml

```diff
@@ -0,0 +1,19 @@
+name: "Search"
+description: "Search box"
+fixtures:
+  default: {}
+  for_use_on_a_dark_but_not_black_background:
```

Comment:
Would this be better as `on_govuk_blue`? That's the manuals use case?
The dark background parameter might suggest it is ok to use on black.